docs/cloud/cloud-databases/cloud-db-serverless-scale.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Scale a serverless database
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 3
+---
+
+# How do I scale a serverless up or down database?
+{: .no_toc }
+
+{% include /serverless/serverless-preview-warning.md %}
+
+This procedure explains how to scale a serverless database up or down. 
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+{% include /serverless/serverless-worker-shape.md %}
```

Comment:
You need to have already created a Serverless database to scale one.

```
* [Learn about FeatureBase Cloud Serverless Databases](/docs/cloud/cloud-databases/cloud-db-serverless-home)
* [Learn how to create a Serverless Database](/docs/cloud/cloud-databases/cloud-db-serverless-create)

Reply:
Plus rather than writing out the instructions to open SQL editor, just link to it

```
* [Learn how to write queries in the FeatureBase Cloud](/docs/cloud-query/cloud-query-data)
```